samples/cellular/modem_shell/sample.yaml

```diff
@@ -290,6 +290,13 @@ tests:
       - thingy91/nrf9160/ns
       - thingy91x/nrf9151/ns
     tags: ci_build
+  sample.cellular.modem_shell.thingy91x_wifi:
+    build_only: true
+    platform_allow:
+      - thingy91x/nrf9151/ns
```

Comment:
Should we add thingy91x compilation for some other basic configurations as well? I mean mainly the default configuration and maybe some other config that might be tight from memory usage perspective.

Reply:
Yes, I'll add `thingy91x` to `sample.cellular.modem_shell.thingy91` and `sample.cellular.modem_shell.thingy91_modem_uart_trace` configurations.

As discussed offline, we could have a look if the `integration_platforms` lists are needed at all. Usually, both `integration_platforms` and `platforms_allow` lists contain the same platforms. I'm not sure `integration_platforms` is needed in this case, as the configuration should be built by default for all allowed platforms.